java/src/main/java/org/rocksdb/TypeUtil.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+package org.rocksdb;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Util class to get internal key for seeking from user key.
+ */
+public class TypeUtil {
+  public static byte[] getInternalKey(byte[] userKey, Options options) {
```

Comment:
Parameters should be marked `final`. (and methods below)

Reply:
done